packages/fill-scenarios/src/index.ts

```diff
@@ -193,11 +212,34 @@ export class FillScenarios {
         ]);
     }
     private async _increaseBalanceAsync(tokenAddress: string, address: string, amount: BigNumber): Promise<void> {
-        await this._zeroEx.token.transferAsync(tokenAddress, this._coinbase, address, amount);
+        const token = new TokenContract(
+            artifacts.Token.abi,
+            tokenAddress,
+            this._provider,
+            this._web3Wrapper.getContractDefaults(),
+        );
+        await token.transfer.sendTransactionAsync(address, amount, {
+            from: this._coinbase,
+        });
     }
     private async _increaseAllowanceAsync(tokenAddress: string, address: string, amount: BigNumber): Promise<void> {
-        const oldMakerAllowance = await this._zeroEx.token.getProxyAllowanceAsync(tokenAddress, address);
+        const tokenInstance = new TokenContract(
+            artifacts.Token.abi,
+            tokenAddress,
+            this._provider,
+            this._web3Wrapper.getContractDefaults(),
+        );
+        const txData = {};
```

Comment:
Remove

Reply:
It is used further down. Moved it closer to where it's used.